src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends AbstractFigure {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, it seems that `AbstractFigure` is being used to share behavior among different figures. Consider using an interface to define common behavior and use composition for shared implementations if necessary.